users/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.urls import path
+from users.views import KakaoSignInView
+
+urlpatterns = [
+    path('/kakao_signin', KakaoSignInView.as_view()),
```

Comment:
여기서는 uri 를 정하실때 언더바 `_` 로 구분하시는 것 보다 `/` 로 구분하시는게 더 좋아보이네요!
```suggestion
    path('/kakao/signin', KakaoSignInView.as_view()),
```

Reply:
넵 수정했습니다!